tests/modules/transdecoder/longorf/test.yml

```diff
@@ -0,0 +1,22 @@
+- name: transdecoder longorf test_transdecoder_longorf
+  command: nextflow run tests/modules/transdecoder/longorf -entry test_transdecoder_longorf -c tests/config/nextflow.config
+  tags:
+    - transdecoder
+    - transdecoder/longorf
+  files:
+    - path: output/transdecoder/pipeliner.3429110.cmds
+      md5sum: abf07276142f8372228e799ae746ab06
+    - path: output/transdecoder/test.__checkpoints_longorfs/TD.longorfs.ok
+      md5sum: d41d8cd98f00b204e9800998ecf8427e
+    - path: output/transdecoder/test.__checkpoints_longorfs/base_freqs_file.ok
+      md5sum: d41d8cd98f00b204e9800998ecf8427e
+    - path: output/transdecoder/test/base_freqs.dat
+      md5sum: 5226ef383532e9b16c16b03e35ce181e
+    - path: output/transdecoder/test/longest_orfs.cds
+      md5sum: 041ac609e4c4ffb38676b1f88365be0f
+    - path: output/transdecoder/test/longest_orfs.gff3
+      md5sum: 96062ef88364f6bf1368d65cd1aad350
+    - path: output/transdecoder/test/longest_orfs.pep
+      md5sum: 0b903d36040676efeb2c50ab5ba46bbd
+    - path: output/transdecoder/versions.yml
+      contains: '[ # TODO nf-core: file md5sum was variable, please replace this text with a string found in the file instead ]'
```

Comment:
You should do as the message says, i.e. check  `versions.yml` for a string that can be checked for and place that here instead of the current string.